backend/fleet_management/migrations/0023_user_last_seen.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.10 on 2020-04-19 22:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fleet_management', '0022_auto_20191221_2303'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='last_seen',
+            field=models.DateTimeField(blank=True, null=True),
```

Comment:
```suggestion
            field=models.DateTimeField(blank=True, null=False),
```
I suggest avoiding nulls overall in database design. `blank` works fine where we don't care that much about the value.